src/pages/index.js

```diff
@@ -1,42 +1,35 @@
 // JS imports
+import { settings } from "../utils/constants.js";
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
```

Comment:
Please, delete `this._toggleButtonState();` from line 23 in `FormValidator`

You need to **disable** the button only in a `then` block after a **successful response** to be able to click it again if there is a server error.